addons/yatl/event/event.gd

```diff
@@ -1,9 +1,10 @@
-extends Resource
+extends Node
```

Comment:
Is there a reason for it being a Node other than using the `process` loop?

Reply:
I'm contemplating replacing the process loop with a timer, so that could be created as a part of PAL.

Reply:
In that case the timer would need to be managed by PAL as well, since anything below Node in inheritance does not have any sense of time, nor can it get the SceneTree to make a SceneTreeTimer.